web/frontend/src/components/camera.vue

```diff
@@ -28,18 +34,31 @@ let pcdExpanded = $ref(false);
 let pointcloud = $ref<Uint8Array | undefined>();
 let camera = $ref(false);
 
-const selectedValue = ref('live');
+const selectedValue = ref('Live');
 
 const initStreamState = () => {
   cameraStreamStates.set(props.cameraName, false);
 };
 
+const clearStreamContainer = (camName: string) => {
```

Comment:
Copied from app.vue. Not happy that it's just modifying the DOM.

Reply:
Agreed, looks fine for now but I'll make a ticket for removing all direct DOM manipulation from RDK.